service/messageHandler_test.go

```diff
@@ -234,9 +247,21 @@ func TestPredicateValidation(t *testing.T) {
 	actualAnnotations := actualBody.Annotations
 	assert.Len(t, actualAnnotations, 1, "annotations")
 	assert.Equal(t, brandUuid, actualAnnotations[0].Concept.ID, "brand annotation")
+
+	// check of monitoring logging
```

Comment:
Could these monitoring logging checks be refactored out?

Reply:
Makes absolutely sense!